pom.xml

```diff
@@ -1353,15 +1363,15 @@
     <profile>
       <id>cdh5</id>
       <activation>
+        <activeByDefault>true</activeByDefault>
         <property>
           <name>hadoop.profile</name>
           <value>cdh5</value>
         </property>
       </activation>
-      <properties>
-        <!-- test cdh5 with the same jars that were produced for default profile -->
-        <maven.main.skip>true</maven.main.skip>
-      </properties>
+      <modules>
+        <module>kite-maven-plugin</module>
```

Comment:
We don't use this module at all. I thought about pulling it out entirely, but this seemed like a good middle ground. I'll add a comment.